src/plugin/__tests__/element.spec.ts

```diff
@@ -654,28 +662,107 @@ describe("buildElementPlugin", () => {
             element.values.nestedElementValues;
           }
         });
-      });
-      it("should output any errors", () => {
-        const {
-          insertElement,
-          getElementDataFromNode,
-          view,
-          serializer,
-        } = createEditorWithElements({
-          testElementWithValidation,
-        });
 
-        insertElement({
-          elementName: "testElementWithValidation",
-          values: { field1: "Some text" },
-        })(view.state, view.dispatch);
-
-        const element = getElementDataFromNode(
-          view.state.doc.firstChild as Node,
-          serializer
-        );
+        describe("elementValidator", () => {
+          it("should output found errors", () => {
+            const {
+              insertElement,
+              getElementDataFromNode,
+              elementDataValidator,
+              view,
+              serializer,
+            } = createEditorWithElements({
+              testElementWithValidation,
+              testElementWithDifferentValidation,
+            });
+
+            insertElement({
```

Comment:
Super minor: do we need to round trip this data through a node and out again? Could be a smaller test if we just pass object values directly. (We could also make this change in the test below.)

Reply:
I think I might add a specific test case for this, to ensure that we can always pass the output from `getElementDataFromNode` directly to the validator. Unless you see this as redundant?

Reply:
It just expands the surface area of what we're testing, but I see the intent – no strong opinions 👍